cache/modules/cloudfront_policies/cache_policies.tf

```
@@ -0,0 +1,111 @@
+resource "aws_cloudfront_cache_policy" "toggle_cookies_only" {
+  name    = "toggle-cookies-only"
+  comment = "Caches based on toggle cookies and nothing else"
+
+  min_ttl     = 0
+  default_ttl = local.one_hour
+  max_ttl     = local.one_day
+
+  parameters_in_cache_key_and_forwarded_to_origin {
+    cookies_config {
+      cookie_behavior = "whitelist"
+
+      cookies {
+        items = local.toggles_cookies
+      }
+    }
+
+    headers_config {
+      header_behavior = "none"
+    }
+
+    query_strings_config {
+      query_string_behavior = "none"
+    }
+  }
+}
+
+resource "aws_cloudfront_cache_policy" "static_content" {
+  name    = "static-content"
+  comment = "Longer-lived cache for static content"
+
+  min_ttl     = 0
+  default_ttl = local.one_day
+  max_ttl     = local.one_year
+
+  parameters_in_cache_key_and_forwarded_to_origin {
+    cookies_config {
+      cookie_behavior = "none"
+    }
+
+    headers_config {
+      header_behavior = "none"
+    }
+
+    query_strings_config {
+      query_string_behavior = "none"
+    }
+  }
+}
+
+resource "aws_cloudfront_cache_policy" "weco_apps" {
+  name    = "weco-apps"
+  comment = "A general-purpose policy for the wc.org apps, including all their usual query strings"
+
+  min_ttl     = 0
+  default_ttl = local.one_hour
+  max_ttl     = local.one_day
+
+  parameters_in_cache_key_and_forwarded_to_origin {
+    cookies_config {
+      cookie_behavior = "whitelist"
+
+      cookies {
+        items = sort(distinct(concat(local.toggles_cookies, local.works_cookies)))
+      }
+    }
+
+    headers_config {
+      header_behavior = "none"
+    }
+
+    query_strings_config {
+      query_string_behavior = "whitelist"
+
+      query_strings {
+        items = sort(distinct(concat(local.content_query_params, local.works_query_params, local.images_query_params)))
```

Might be worth breaking this up across multiple lines for readability, e.g.
```suggestion
        items = sort(
          distinct(
            concat(
              local.content_query_params,
              local.works_query_params,
              local.images_query_params
            )
          )
        )
```